### ISSUE TITLE
Focus Order for Photons and Layer Model screens

### ISSUE THREAD

**Issue by arouinfar** [phetsims/greenhouse-effect#221]
Related to #220. Alt input is technically working on the Photons and Layer Model screens, but the focus order is currently jumbled which makes the sim difficult to use. @jbphet please let me know if you have any questions.

**Photons Screen**
The Photons screen should have the same focus order as the Waves screen. The only additions are the Flux Meter checkbox and slider, which should slot in like this:

- Energy Balance Checkbox
- Flux Meter Checkbox
- Flux Meter Slider
- Cloud Checkbox

**Layer Model Screen**
* Start Sunlight Button
* Control panels working top-down:
   * Solar Intensity Slider
   * Surface Albedo Slider
   * Absorbing Layers NumberPicker
   * Infrared Absorbance Slider
* Temperature readouts working bottom-up (matches the order the layers are added)
   * Surface temperature checkbox
   * Layer 1 temperature checkbox (first layer added, closest to ground)
   * Layer 2 temperature checkbox
   * Layer 3 temperature checkbox (last layer added, farthest from ground)
* Energy Balance checkbox
* Flux Meter checkbox
* Flux Meter slider
* Zoom out button
* Zoom in button
* Controls across the bottom of the screen working top-down, left-right:
   * Temperature Units RadioButtonGroup
   * More Photons checkbox
   * PlayPauseButton
   * StepForwardButton
   * Normal/Slow RadioButtonGroup
   * ResestAllButton

**Comment by jbphet** [comment]
This is now implemented.  @arouinfar - please check it over and let me know if you see any problems.

**Comment by arouinfar** [comment]
Thanks @jbphet! Everything looks good in master, closing.
